charts/kyverno/templates/_helpers/_pdb.tpl

```diff
@@ -1,7 +1,7 @@
 {{/* vim: set filetype=mustache: */}}
 
 {{- define "kyverno.pdb.apiVersion" -}}
-{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1" -}}
```

Comment:
`policy/v1/PodDisruptionBudget` looks correct to me. Why are you removing the kind here ?

Reply:
the correct APIVersion is `policy/v1` 
`PodDisruptionBudget` is a resource within the `policy/v1` api.

Reply:
From the helm docs:

> Capabilities.APIVersions.Has $version indicates whether a version (e.g., batch/v1) or resource (e.g., apps/v1/Deployment) is available on the cluster.

`policy/v1/PodDisruptionBudget` should be supported, are you trying to address a specific issue ?

Reply:
@r2ronoha @kuldazbraslav are you still working on this ?

Reply:
Hi, yes

> are you trying to address a specific issue
the fact that using `policy/v1/PodDisruptionBudget` does not set the right apiVersion. 

We are currently running a custom version of the chart using the suggested change to work around it

Reply:
Can you elaborate ?
Are you using a specific tool ? Or just helm ?

Reply:
@r2ronoha please give us more details on your setup, the behaviour you are observing seems to contradict with helm docs.

Reply:
Are you using ArgoCD ?